packages/server/safeMutations/acceptTeamInvitation.ts

```diff
@@ -112,9 +106,6 @@ const acceptTeamInvitation = async (
     await setUserTierForUserIds([userId])
   }
 
-  // if a meeting is going on right now, add them
-  await addTeamMemberToMeetings(teamMember, teamId, dataLoader)
```

Comment:
joining a meeting when joining a team always felt bad. feels good doing this lazily, when the user joins the meeting